<html>
    <head>

    </head>
    <body>
        <button onclick="throwError()">throw error</button>

<script>
    function throwError(){
        try {
            attemptDivision();
        }catch (error) {
            console.log(error.message+" - Error Type: "+ error.name);
        }
    }

function attemptDivision(){
    let result;

    try {
        result= x / 10;
    } catch (error) {
        //Always include at least a 'message' and 'name' properties
        throw{
            "message": "In the attempDivision() method the following error occurres: " +error.message,
            "name":"CustomError"
        };
    }
}
    
</script>

        <script src="jquery-1.11.2.js"></script>
        <script src="Greetr.js"></script>
        <script src="app.js"></script>
    </body>
</html>